charts/kueue/README.md

```diff
@@ -67,3 +67,4 @@ The following table lists the configurable parameters of the kueue chart and the
 | `managerConfig.controllerManagerConfigYaml`            | controllerManagerConfigYaml                            | abbr.                                       |
 | `metricsService`                                       | metricsService's ports                                 | abbr.                                       |
 | `webhookService`                                       | webhookService's ports                                 | abbr.                                       |
+| `apf.version`                                          | override flowcontrol.apiserver.k8s.io version          | v1                                          |
```

Comment:
Instead of this option, could you use the Capability.KubeVersion?
https://helm.sh/docs/chart_template_guide/builtin_objects/

Reply:
Done